src/common/gamedata.rs

```diff
@@ -291,3 +317,10 @@ impl GameData {
         self.get_weather_rate(*weather_rate_id as u32)
     }
 }
+
+// Simple enum for GameData::get_territory_name
+pub enum TerritoryNameKinds {
```

Comment:
```suggestion
pub enum TerritoryNameKind {
```

Small nitpick, it would make sense for it to be plural if it was flag (so kinds could be a combination)